### PR TITLE
Feature/get revision

### PR DESCRIPTION
**Pull request by Sandychuang8**
Make document's revision id visible in API.

### CODE REVIEW

File: src/Couchbase.Lite.Shared/API/Document/Document.cs [hunk 1]
```diff
@@ -112,8 +112,14 @@ internal C4DocumentWrapper c4Doc
...
 
+        /// <summary>
+        /// The RevisionID in Document class is a constant, while the RevisionID in MutableDocument class is not. Every time a MutableDocument is saved, 
+        /// its internal revision (via C4Document) will be updated. When a new MutableDocument is created, the document will not have C4Document 
+        /// nor Revision associated with it, therefore, the revisionID will be null.
+        /// The RevisionID format is opaque, which means it's format has no meaning and shouldn�t be parsed to get information.
+        /// </summary>
         [CanBeNull]
-        internal string RevID => c4Doc?.HasValue == true ? c4Doc.RawDoc->selectedRev.revID.CreateString() : null;
+        public string RevisionID => c4Doc?.HasValue == true ? c4Doc.RawDoc->selectedRev.revID.CreateString() : null;
```
Comment (borrrden) [inline src/Couchbase.Lite.Shared/API/Document/Document.cs]: Leave out the stuff explaining about `C4Document`.  It is internal and the developer doesn't need to worry about it.  Just note that a newly created document will have a `null` RevisionID and that mutable documents will be updated on save.  Also, use `<see cref="MutableDocument" />` when you talk about mutable document so that it creates a clickable link in the API docs.

File: src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj [hunk 2]
```diff
@@ -83,6 +83,9 @@
     <EmbeddedNativeLibrary Include="..\..\vendor\couchbase-lite-core\build_cmake\android\lib\x86\libLiteCore.so">
       <Link>x86\libLiteCore.so</Link>
     </EmbeddedNativeLibrary>
+    <EmbeddedNativeLibrary Include="..\..\vendor\couchbase-lite-core\build_cmake\android\lib\x86_64\libLiteCore.so">
+      <Link>x86_64\libLiteCore.so</Link>
+    </EmbeddedNativeLibrary>
```
Comment (borrrden) [inline src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj]: This is not part of this PR.  Let's handle it separately.

File: src/build/do_fetch_litecore.ps1 [hunk 3]
```diff
@@ -13,14 +13,15 @@ if($DebugLib) {
 
 Write-Host "Fetching variants for $Sha..."
 if($Variants[0].ToLower() -eq "all") {
-    $Variants = @("macosx", "linux", "ios", "android-x86", "android-armeabi-v7a", "android-arm64-v8a", "windows-win32", "windows-win64", "windows-arm")
+    $Variants = @("macosx", "linux", "ios", "android-x86", "android-x86_64", "android-armeabi-v7a", "android-arm64-v8a", "windows-win32", "windows-win64", "windows-arm")
```
Comment (borrrden) [inline src/build/do_fetch_litecore.ps1]: None of this stuff is relevant to this PR.

File: src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj (hunk 2 above)
Comment (Sandychuang8) [reply src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj]: Not sure why it's appeared here...

File: src/build/do_fetch_litecore.ps1 (hunk 3 above)
Comment (Sandychuang8) [reply src/build/do_fetch_litecore.ps1]: Not sure why it's appeared here...

File: src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj (hunk 2 above)
Comment (borrrden) [reply src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj]: Check your commit in SourceTree.  Make sure it doesn't have anything you don't recognize.  It looks like you pulled the cobalt branch into the master branch maybe?

Comment (Sandychuang8) [reply src/Couchbase.Lite.Support.Android/Couchbase.Lite.Support.Android.csproj]: yes, that's what I did..
